modules/ensemble/lib/widget/loading_container.dart

```diff
@@ -148,42 +142,45 @@ class LoadingContainerState extends EWidgetState<LoadingContainer> {
   }
 
   LinearGradient _buildGradient() {
+    List<Color> colors = widget._controller.gradientColors ??
+        [
+          widget._controller.baseColor ?? const Color(0xFFEBEBF4),
+          widget._controller.highlightColor ??
+              const Color(0xFFEBEBF4).withOpacity(0.3),
+          widget._controller.baseColor ?? const Color(0xFFEBEBF4),
+        ];
+
+    return LinearGradient(
+      begin: _getGradientBegin(),
+      end: _getGradientEnd(),
+      colors: colors,
+      stops: widget._controller.gradientStops ?? const [0.1, 0.3, 0.4],
+      tileMode: TileMode.clamp,
```

Comment:
this should not be hardcoded. Default could be clamp but give the user options to pick something else